infrastructure/pcluster-manager.yaml

```diff
@@ -38,10 +38,28 @@ Parameters:
     Type: String
     Default: ''
   OIDCProvider:
-    Description: OICD provider to use (defaults to Amazon Cognito).
+    Description: OIDC provider to use (defaults to Amazon Cognito).
     Type: String
-    AllowedValues: [Cognito]
+    AllowedValues: [Cognito, Okta]
     Default: Cognito
+  OktaDomain:
+    Description: The Okta domain.
+    Type: String
+    Default: ''
+  OktaServerId:
+    Description: The ID of the Okta authorization server.
+    Type: String
+    Default: ''
+  OktaClientSecret:
+    Description: The client secret.
+    Type: String
+    NoEcho: true
+    Default: ''
```

Comment:
issue(blockgin): as said in the comment below, we also need OktaClientId here (it could be NoEcho: false since it's not super sensible)